test/controllers/TrusteeAUKCitizenControllerSpec.scala

```diff
@@ -0,0 +1,188 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.SpecBase
+import forms.TrusteeAUKCitizenFormProvider
+import models.{FullName, NormalMode, UserAnswers}
+import navigation.{FakeNavigator, Navigator}
+import pages.{TrusteeAUKCitizenPage, TrusteesNamePage}
+import play.api.inject.bind
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import views.html.TrusteeAUKCitizenView
+
+
+class TrusteeAUKCitizenControllerSpec extends SpecBase {
+
+  def onwardRoute = Call("GET", "/foo")
+
+  val formProvider = new TrusteeAUKCitizenFormProvider()
+  val form = formProvider()
+
+  val index = 0
+  val emptyTrusteeName = ""
+  val trusteeName = "FirstName LastName"
+
+  lazy val trusteeAUKCitizenRoute = routes.TrusteeAUKCitizenController.onPageLoad(NormalMode, index).url
+
+  "trusteeAUKCitizen Controller" must {
+
+    "return OK and the correct view for a GET" in {
+
+      val userAnswers = UserAnswers(userAnswersId)
+        .set(TrusteesNamePage(index), FullName("FirstName", None, "LastName")).success.value
+
+      val application = applicationBuilder(userAnswers = Some(userAnswers)).build()
+
+      val request = FakeRequest(GET, trusteeAUKCitizenRoute)
+
+      val result = route(application, request).value
+
+      val view = application.injector.instanceOf[TrusteeAUKCitizenView]
+
+      status(result) mustEqual OK
+
+      contentAsString(result) mustEqual
+        view(form, NormalMode, index, trusteeName)(fakeRequest, messages).toString
+
+      application.stop()
+    }
+
+    "populate the view correctly on a GET when the question has previously been answered" in {
+
+      val userAnswers = UserAnswers(userAnswersId)
+        .set(TrusteesNamePage(index), FullName("FirstName", None, "LastName")).success.value
+        .set(TrusteeAUKCitizenPage(index), true).success.value
+
+      val application = applicationBuilder(userAnswers = Some(userAnswers)).build()
+
+      val request = FakeRequest(GET, trusteeAUKCitizenRoute)
+
+      val view = application.injector.instanceOf[TrusteeAUKCitizenView]
+
+      val result = route(application, request).value
+
+      status(result) mustEqual OK
+
+      contentAsString(result) mustEqual
+        view(form.fill(true), NormalMode, index, trusteeName)(fakeRequest, messages).toString
+
+      application.stop()
+    }
+
+    "redirect to SessionExpired when TrusteesName is not answered" in {
```

Comment:
This doesn't go to SessionExpired it goes to TrusteesNameController as per your test.